includes/services/extended/pocket.php

```diff
@@ -18,7 +18,9 @@ function __construct() {
 		$this->set_endpoint( 'access_token', 'https://getpocket.com/v3/oauth/authorize', 'POST' );
 
 		$creds              = $this->get_credentials();
-		$this->key          = $creds['key'];
+		if ( ! empty( $creds ) ) {
```

Comment:
Maybe this should be: 
```suggestion
		if ( ! empty( $creds['key'] ) ) {
```

So we make sure the key exists too?